.github/workflows/static_build.yml

```diff
@@ -1,5 +1,4 @@
 name: Micromamba
-
```

Comment:
I think we should keep the space here for consistency with the rest of the yaml files 🙏 (must be the linter I guess).